content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,178 @@
+[here](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/env/gcp) for more details. 
+
+### Change deploy namespace
+To deploy Kubeflow Pipelines in namespace FOO,
+- Edit [dev/kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/env/dev/kustomization.yaml)
+or [gcp/kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/env/gcp/kustomization.yaml)
+namespace section to FOO
+- Then run 
+```
+kubectl apply -k manifests/kustomize/env/dev # or other env of your choice
+```
+
+### Disable the public endpoint
+By default, the deployment install an [invert proxy agent](https://github.com/google/inverting-proxy) that exposes a public URL. If you want to skip installing it,
+- Comment out the proxy component in the [kustomization.yaml](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/kustomization.yaml).
+- Then run 
+```
+kubectl apply -k manifests/kustomize/env/dev # or other env of your choice
+```
+
+The UI is still accessible by port-forwarding
+```
+kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+```
+and open http://localhost:8080/
+
+
+
+## Uninstall
+You can uninstall Kubeflow Pipelines by running
+```
+export PIPELINE_VERSION={{% kfp-latest-version %}}
+kubectl delete -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+```
+
+Or if you deployed through kustomize
+```
+kubectl delete -k manifests/kustomize/env/dev
+# or
+kubectl delete -k manifests/kustomize/env/gcp
+```
+
+## Best practice maintaining custom manifests
+
+### Maintain a repo for your manifests
+
+Save the following to a source controlled repo.
+
+File `kustomization.yaml`.
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Edit the following to change the deployment to your custom namespace.
+namespace: kubeflow
+
+# You can add other customizations here using kustomize.
+
+# Edit ref in the following link to deploy a different version of KFP.
+bases:
+- github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref={{% kfp-latest-version %}}
+```
+
+### How to?
+* Deploy: `kubectl apply -k $YOUR_REPO`
+* Upgrade
+    1. (Recommended) backup your data storages for KFP.
+    1. Edit `ref={{% kfp-latest-version %}}` to a version you want to upgrade to.
+    Check [Kubeflow Pipelines github repo](https://github.com/kubeflow/pipelines/releases) for available releases.
+    1. Deploy: `kubectl apply -k $YOUR_REPO`
+* Downgrade
+    1. Make sure you made a backup before upgrading.
+    1. Delete the deployment `kubectl delete -k $YOUR_REPO`.
+    1. Edit `kustomization.yaml` to use backup storage.
+    1. Deploy: `kubectl apply -k $YOUR_REPO`.
+
+
+### Further reading
+kustomize's [recommended workflow using an Off-The-Shelf configuration](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/workflows.md#off-the-shelf-configuration)
+like KFP lite we provide.
```

Comment:
It's not clear what this means: "like KFP lite we provide." Please clarify the wording.